interfaces/IBrowser.h

```diff
@@ -149,6 +161,17 @@ namespace Exchange {
 
         // @brief Initiate garbage collection
         virtual uint32_t CollectGarbage() = 0;
+        // @brief Delete directory
+	// @param path: Path to be deleted
+        virtual uint32_t Delete(const string& path) = 0;
```

Comment:
This on JSONRPC is not a very good option. Did you find it somewhere in *.json file ?